src/bin/ws_orders.rs

```diff
@@ -0,0 +1,35 @@
+use log::info;
+
+use std::str::FromStr;
+
+use ethers::types::H160;
+use hyperliquid_rust_sdk::{BaseUrl, InfoClient, Message, Subscription};
+use tokio::{
+    spawn,
+    sync::mpsc::unbounded_channel,
+    time::{sleep, Duration},
+};
+
+#[tokio::main]
+async fn main() {
+    env_logger::init();
+    let mut info_client = InfoClient::new(None, Some(BaseUrl::Testnet)).await.unwrap();
+    let user = H160::from_str("0x4cd1B55373c9259eA927F8a489D866CDaF88a0e9").unwrap();
```

Comment:
nit: since this might be a personal address, you may want to make this a placeholder address and add a comment for the user to fill in their address, or expose it as a CLI arg

Reply:
It was a random wallet I created to test this, but I changed it to the same as the ws_users example